hgtector/search.py

```diff
@@ -131,6 +131,14 @@ def __call__(self, args):
         # read and validate input data
         self.input_wf()
 
+	# check requested threads if set vs available ones.
+        m=len(sched_getaffinity(0))
```

Comment:
Please add spaces around the variable: `m = len(sched_getaffinity(0))`. (I try to make the code PEP8 compliant.)